tool/doc.dart

```diff
@@ -95,8 +96,7 @@ late Map<String, SinceInfo> sinceInfo;
 
 final Map<String, String> _fixStatusMap = <String, String>{};
 
-String describeMaturity(LintRule r) =>
-    r.maturity == Maturity.stable ? '' : ' (${r.maturity.name})';
+String describeMaturity(LintRule r) => r.state.isStable ? '' : r.state.label;
```

Comment:
Consider renaming this to `describeState`.

Reply:
Done. Thanks!